### PR TITLE
new icon: langchain (original, original-wordmark)

### PR DESCRIPTION
**Pull request by kimjiwook0129**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [x] A new object is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

**This PR closes NONE**

### CODE REVIEW

File: icons/langchain/langchain-original-wordmark.svg
Comment (Snailedlt) [inline icons/langchain/langchain-original-wordmark.svg:1]: This (and the original) icon is very large. Please look into optimizing it. If you don't know how, take a look at this guide: https://github.com/devicons/devicon/issues/1306#issuecomment-1296015861
There are many other guides also under that issue that might be worth looking at. For example this one for creating line versions from plain or original versions: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294

**Review by Snailedlt — CHANGES_REQUESTED** [review]

File: devicon.json [hunk 1]
```diff
@@ -5583,6 +5583,35 @@
...
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
```
[the remark this answers is not in the document]
Comment (kimjiwook0129) [reply devicon.json]: I've optimized both icons based on [#1306 (comment)](https://github.com/devicons/devicon/issues/1306#issuecomment-1296015861).
Could you recommend a tool I can use to make the svgs mono-colored?

[the remark this answers is not in the document]
Comment (Snailedlt) [reply devicon.json]: @kimjiwook0129 Great!
We don't have any guides for making plain icons afaik, but I recommend trying to make it using inkscape.

Alternatively you could make a line icon using this guide: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294

If it's okey with you I'll review the PR once you've added a font supported version (plain or line) with and without wordmark :)

[the remark this answers is not in the document]
Comment (kimjiwook0129) [reply devicon.json]: I've followed the guide using InkScape and attempted to convert the icons to outlined paths. However, at stage 7 of [#1306 (comment)](https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294), I encountered an issue. 

We have approximately 300 folders containing both the stroke (as a path) and the initial path at the end of stage 6, and each initial path and folder needs to be manually removed while moving each stroke to the root of the SVG. 

I managed to handle around 100 folders, but it has proven to be quite challenging. There doesn't seem to be an automatic feature to drag all the strokes to the root of the SVG at once afaik. 

It appears that the current icons I am using do not meet the devicon guidelines. 
If it is acceptable to you, we can close this PR, and I will revisit it later with a suitable icon.

**Pull request closed without merging**